packages/types/src/discord/guild.ts

```diff
@@ -681,11 +703,12 @@ export interface DiscordPrunedCount {
 
 /** https://discord.com/developers/docs/resources/guild#get-guild-vanity-url */
 // TODO: This should provably have another name for clarity to what it really rappresent
-export interface DiscordVanityUrl {
+export interface DiscordVanityUrl extends Partial<Omit<DiscordInvite, 'code'>> {
   code: string | null
```

Comment:
```suggestion
export interface DiscordVanityUrl extends Partial<DiscordInvite> {
code: string | null
```
Technically with interface, you don't need to Omit 'code', since it will be overriden by the `code: string | null` anyways, but both way is fine.

Reply:
You actually get an error for this:

```
Interface 'DiscordVanityUrl' incorrectly extends interface 'Partial<DiscordInvite>'.
  Types of property 'code' are incompatible.
    Type 'string | null' is not assignable to type 'string | undefined'.
      Type 'null' is not assignable to type 'string | undefined'.ts(2430)
```